SpatialGDK/Documentation/features.md

```diff
@@ -32,13 +32,13 @@ The SpatialOS Unreal GDK also supports:
 * Replicated Blueprint events
 * RPC validation
 
-## Static subobject replication
+## Subobject replication
 
-The SpatialOS Unreal GDK currently supports only one instance of each static subobject type when replicating objects owned by an Actor. This includes components. For example, multiple Actors can have a `FooComponent` but no Actor can have two `FooComponent`s.
+The GDK currently supports only one instance of each static subobject type when replicating objects owned by an Actor. This includes components. For example, multiple Actors can have a `FooComponent` but no Actor can have two `FooComponent`s.
 
 Properties and RPCs within subobjects have the same support as an Actor’s properties and RPCs.
 
-> The SpatialOS Unreal GDK does not yet support dynamic subobject replication.
+The SpatialOS Unreal GDK does not yet support dynamic subobject replication.
```

Comment:
SpatialOS Unreal GDK -> SpatialOS GDK for Unreal Engine

Reply:
Good spot!
"SpatialOS Unreal GDK"
->
"GDK" 

[Suggest going for the shorter name as we are in the GDK docs - it's easier to read - as Giray said about a year ago! (He was right!)]